### PR TITLE
logging and notes added to recipe-form.js

### PR DESCRIPTION
**Pull request by hammypwns**

### CODE REVIEW

File: src/components/recipe-form.js [hunk 1]
```diff
@@ -136,13 +138,41 @@ const RecipeForm = ({
...
+    }
+    if (image_url != values.image_url) {
+      log += 'Image, '
+    }
+
+    //remove trailing ', ' if present
+    if (log != '') {
+      log = log.substring(0, log.length - 2)
+    }
+
```
Comment (garrowp) [inline src/components/recipe-form.js:167]: One thought on this. Do we want to save the log as a big string or as an array? Either way, if `log` was an array at first, we could `push` all of the values into `log` and then just do a `.join(',')` at some point to turn it into a string with commas in between. Then we don't have to remove the trailing comma. Just a thought

File: src/components/image-dropzone.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const ImageDropZone = ({ handleImageDrop, image_url, name }) => {
   const alt = file ? file.name : name
 
   const thumb =
-    src.length > 0 ? (
+    src && src.length > 0 ? (
```
Comment (garrowp) [inline src/components/image-dropzone.js:22]: Did this actually work for anything? If not, we should remove it for now and figure out the image stuff later

**Review by garrowp — COMMENTED** [review]
These changes require a change in the db, correct? Are those changes already reflected in our dev environment?

**Review by garrowp — APPROVED** [review]

File: src/components/image-dropzone.js (hunk 2 above)
Comment (hammypwns) [reply src/components/image-dropzone.js:22]: it fixed buggy bois

File: src/components/recipe-form.js (hunk 1 above)
Comment (hammypwns) [reply src/components/recipe-form.js:167]: yeet

**Pull request merged**